utils/healthcheck/healthcheck_ping_manager.py

```diff
@@ -0,0 +1,61 @@
+from datetime import datetime
+
+import aiohttp
+
+from common.configuration import (
+    HEALTHCHECK_DATA_COLLECTOR_URL,
+    HEALTHCHECK_PING_ENABLED,
+    HEALTHCHECK_PING_INTERVAL,
+    HEALTHCHECK_SERVICE_API_KEY,
+)
+from common.logging import get_logger
+
+logger = get_logger()
+
+
+class HealthcheckPingManager:
+    def __init__(self) -> None:
+        self.targets = {
+            "data_collector": {
```

Comment:
Do we expect any expansion on this healthcheck manager?

Reply:
I just preferred to create it in a expansible way, but I don't expect this to be needed in the short-term.